content/docs/faq-build.md

```diff
@@ -1,32 +1,32 @@
 ---
 id: faq-build
-title: Babel, JSX, and Build Steps
+title: Babel, JSX, dan Tahapan Build
 permalink: docs/faq-build.html
 layout: docs
 category: FAQ
 ---
 
-### Do I need to use JSX with React? {#do-i-need-to-use-jsx-with-react}
+### Perlukah saya menggunakan JSX pada React? {#do-i-need-to-use-jsx-with-react}
 
-No! Check out ["React Without JSX"](/docs/react-without-jsx.html) to learn more.
+Tidak! Silahkan lihat ["React Tanpa JSX"](/docs/react-without-jsx.html) untuk lebih lanjut.
 
-### Do I need to use ES6 (+) with React? {#do-i-need-to-use-es6--with-react}
+### Perlukah saya menggunakan ES6 (+) pada React? {#do-i-need-to-use-es6--with-react}
```

Comment:
I think for the context of this sentence `dengan` is a closer translation than `pada`
What do you think? 😄